Fixes error:
drivers/net/wireless/Kconfig:289: can't open file "drivers/net/wireless/sprdwl/Kconfig"
diff --git a/drivers/net/wireless/Kconfig b/drivers/net/wireless/Kconfig
index bcc52541..23931336 100644
--- a/drivers/net/wireless/Kconfig
+++ b/drivers/net/wireless/Kconfig
@@ -286,7 +286,7 @@ source "drivers/net/wireless/rtlwifi/Kconfig"
 source "drivers/net/wireless/ti/Kconfig"
 source "drivers/net/wireless/zd1211rw/Kconfig"
 source "drivers/net/wireless/mwifiex/Kconfig"
-source "drivers/net/wireless/sprdwl/Kconfig"
+#source "drivers/net/wireless/sprdwl/Kconfig"
 #source "drivers/net/wireless/sc2331/Kconfig"
 source "drivers/net/wireless/sc2331/sc2331_mem_alloc/Kconfig"
 endif # WLAN
